File: labs/lab07/total_words.py
#!/usr/bin/python

import sys
import re

words = []

# loop through stdin
for lines in sys.stdin:
	words += re.split(r'[^a-zA-Z]', lines);

#filder words
words = filter(None, words);
# print output
print str(len(words)) + " words";

File: labs/lab07/total_words.pl
#!/usr/bin/perl

$total_words = 0;
while ( ($line = <STDIN>) ) {
      	chomp $line;
	@words = grep(/./, split(/[^a-zA-Z]/, $line));
	$total_words += 0+@words;
}

print $total_words, " words\n";
